### PR TITLE
Cult stun nerf

### PR DESCRIPTION
**Pull request by Archanial**
## About The Pull Request

Stun now checks for mindshield and makes mindshielded personnel only drop items, become dizzy, lose about 8% of their blood and take 60 stamina damage (same as one hit with stunbaton).

## Why It's Good For The Game

One hit win is okay if you are trying to convert some random assistant but I don't like the idea of instant one-click win against security. Cult still has really good weapons, it's not like stun was essential to fight off security.
I might've overnerfed it, leave your opinions and ideas in comments.
If it commes out as too weak we can always add some stamina damage.

## Changelog
:cl:
balance: Cult stun no longer stuns mindshielded personnel. Instead it makes them drop items they are currently holding, applies a dizziness effect and does some stamina damage.
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
After months of thread posting, it finally is here

File: code/modules/antagonists/cult/blood_magic.dm [hunk 1]
```diff
@@ -434,19 +434,28 @@
...
-				var/mob/living/silicon/S = L
-				S.emp_act(EMP_HEAVY)
-			else if(iscarbon(target))
+			if(HAS_TRAIT(target, TRAIT_MINDSHIELD))
 				var/mob/living/carbon/C = L
-				C.silent += 6
-				C.stuttering += 15
-				C.cultslurring += 15
-				C.Jitter(15)
+				to_chat(user, "<span class='cultitalic'>Their mind was stronger than exected, but you still managed to do some damage!</span>")
```
Comment (cacogen) [inline code/modules/antagonists/cult/blood_magic.dm:441]:
```suggestion
				to_chat(user, "<span class='cultitalic'>Their mind was stronger than expected, but you still managed to do some damage!</span>")
```

Comment (Archanial) [reply code/modules/antagonists/cult/blood_magic.dm:441]: whoops

**Comment by Shadowflame909** [comment]
touch fuzzy get dizzy

**Comment by wesoda25** [comment]
Better solution than anything I saw offered in the thread, 👍

**Comment by Archanial** [comment]
I actually didn't read the thread, can someone link it here?

**Comment by SkeletalElite** [comment]
I can see this becoming a problem if mass mindshielding happens since cult cant really tell if someone has a mindshield without actually trying this on them. Maybe add a way to tell to the zealot blindfold.

**Comment by Archanial** [comment]
> 
> 
> I can see this becoming a problem if mass mindshielding happens since cult cant really tell if someone has a mindshield without actually trying this on them. Maybe add a way to tell to the zealot blindfold.

If you come to a point where mass mindshielding already happened and you have no cultists you played your cards wrong and probably deserve it.

**Comment by itseasytosee** [comment]
Might I suggest moderate bloodloss for theming? probably not more than 5-10%

**Comment by Tlaltecuhtli** [comment]
mindshields during cult was actually a negative thing as the only thing it prevented was you having fun if you happened to get captured
this seems a good change

**Comment by necromanceranne** [comment]
You mean to say that to capture a security officer requires actually fighting them? By god, the man's done it.

**Comment by ATH1909** [comment]
👍 from me.

**Comment by Megarop** [comment]
if the cult wants something that instantly stuns people unless they have something that security has roundstart that makes it barely do anything they can just break into robotics and print a flash

**Comment by Archanial** [comment]
> 
> 
> Might I suggest moderate bloodloss for theming? probably not more than 5-10%

Good idea.


> i disagree, if a one hit win is unbalanced againt security it's unbalanced against everyone else too
> especially during a conversion gamemode like cult where there's too many antags for the rest of the station to be able to sit back and only have sec officers fight them

Cargo exists and mindshields can be ordered. Additionally every brig start with 8 implants.

> also if the cult wants something that instantly stuns people unless they have something from security that makes it barely do anything they can just break into robotics and print a flash

Flash doesn't mute victims. The early stealth is important for cult as they have to setup their base and don't get raided by security early.

**Comment by bobbahbrown** [comment]
what if you made this stun also apply to people who are antagonists in the round so cult cant wreck other antags as easily

**Comment by AdamElTablawy** [comment]
It should still cultslur security instead of making them stutter. And probably flash, to go along with the dizzy. It'd make it reasonably debilitating, without making it useless against sec.

**Comment by Shadowflame909** [comment]
Thinkin about it more. It's def an overnerf.
 
10 minutes into the round, you've set up the maint cultbase. 

Some security officer who actually does his job (rare I know) and he's roaming around maint. They see a glowing room in maint, and welder into it. OH SHIT A SEC OFFICER SEES YOUR CULT BASE.

You literally weren't prepared for this to happen so early. so you're not in murderbone mode yet, and you just have disabling spells like stun, cuffs, teleport, hallucinate and EMP.

He's darting away into dorms. You follow him, use stun on him. Doesn't do much as he's still darting away. You EMP him, but he's literally so close to sec that this doesn't matter.

To prevent cult start-game from getting cheesed by one decent boy (and to prevent cult from being murderboney at roundstart.) I recommend adding knock-down to the cult stun. That sec officer can still fight his way out, with his disabler/stun baton/flash-bang. But he wont be able to easily run away out of maint and reveal its cult no matter what the cultists do. Would also give them time to gear up and assasinate the man.

**Comment by SkeletalElite** [comment]
I still think zealot blindfold should be able to tell if someone is mind shielded because when Vlad Valid the assistant mind shielding himself "just in case" and getting the cult found because a cultist didn't expect an assistant to have a mind shield would be a little bit infuriating. Its rare but it'll probably happen at least every once in a while.

**Comment by AdamElTablawy** [comment]
@Shadowflame909 there's no reason not to have a cult bola

**Comment by Shadowflame909** [comment]
Cult bola comes with the sword. So your new meta will still be roundstart murder cult compared to just making the stun KD. Making it not escate to that.

**Comment by AdamElTablawy** [comment]
@Shadowflame909 What's the downside?

You still murder sec when you stun them anyway. Sword or no sword.

Nothing has changed, it's just marginally harder because you can actually miss.

With old cult, the officer would walk in and you'd stun talisman him, knife him to death and sac.

New cult, you bola the officer instead and stab him to death.

**Comment by Shadowflame909** [comment]
The downside is cult swords dont fit in backpacks.

So if cult is forced to deal with a sec officer early in the round. They're gonna potentially drop it, revealing the roundtype and forcing them to go war-cult faster.

Or they're gonna have to armor up, and go war cult anyways just to store their sword.

Congrats, one slight change such as this forced cult into war mode territory all because they couldn't KD a lone sec officer and had to weapon up and sword em.

**Comment by CPTANT** [comment]
Hmmmm I am afraid we will see a new meta emerge where the entire crew just gets bagged, holy watered and mindshielded.

**Comment by Megarop** [comment]
> Hmmmm I am afraid we will see a new meta emerge where the entire crew just gets bagged, holy watered and mindshielded.

at least it gives mindshields a purpose during cult besides stopping cultists from being deconverted if you mindshield them too early

**Comment by Shadowflame909** [comment]
That just sounds like revs to me.

**Comment by Archanial** [comment]
> Some security officer who actually does his job (rare I know) and he's roaming around maint. They see a glowing room in maint, and welder into it. OH SHIT A SEC OFFICER SEES YOUR CULT BASE.

To be honest every assistant can spot cult base and just run away before they get stunned, but I get your point and you're right.




> I still think zealot blindfold should be able to tell if someone is mind shielded because when Vlad Valid the assistant mind shielding himself "just in case" and getting the cult found because a cultist didn't expect an assistant to have a mind shield would be a little bit infuriating. Its rare but it'll probably happen at least every once in a while.

I don't like the idea of giving cult sec huds. Implanting yourself is rare and implanting yourself "just in case" is powergaming.
If it commes out as a big issue I will add it in separate pr.



> 
> 
> what if you made this stun also apply to people who are antagonists in the round so cult cant wreck other antags as easily

If you mean other antags should be immune to stun too then this is a bad idea. I don't see reason why they shouldn't get converted too. Seems more like issue with dynamic not with cult. 

> 
> 
> Hmmmm I am afraid we will see a new meta emerge where the entire crew just gets bagged, holy watered and mindshielded.

Doesn't that happen now?



I've added 60 stamina damage, same as stunbaton, so you can chase and mess up officer's face with your cult blades.

**Comment by wesoda25** [comment]
if you make it 60 you'll basically just be making it a stun baton

**Comment by Archanial** [comment]
> 
> 
> if you make it 60 you'll basically just be making it a stun baton

It's one use 60 stamina damage without knockdown.

**Comment by gitbirb** [comment]
this would be good if the nerf only took place after culteyes showed up, which is the part of the round that the cult should be prepared or about to die

**Pull request merged**